qiskit_aer/extensions/snapshot_density_matrix.py

```diff
@@ -15,10 +15,10 @@
 """
 
 from qiskit import QuantumCircuit
-from qiskit.providers.aer.extensions import Snapshot
+from qiskit_aer.extensions import snapshot
 
 
-class SnapshotDensityMatrix(Snapshot):
+class SnapshotDensityMatrix(snapshot.Snapshot):
```

Comment:
Snapshot is used in the code below, which is why the CI is giving an import error.